test/e2e/common_test.go

```diff
@@ -56,6 +57,7 @@ type PodmanTestIntegration struct {
 	Timings             []string
 	TmpDir              string
 	RemoteStartErr      error
+	NetavarkConfigDir   string
```

Comment:
Please rename CNIConfigDir to NetworkConfigDir and only use one field for the config dir. The podman network prune tests have to set a special testdir, otherwise they will cause many flakes.